React/Fabric/Mounting/ComponentViews/Slider/RCTSliderComponentView.mm

```diff
@@ -23,7 +24,7 @@ @interface RCTSliderComponentView () <RCTImageResponseDelegate>
 @end
 
 @implementation RCTSliderComponentView {
-  UISlider *_sliderView;
+  RCTSlider *_sliderView; // TODO(macOS GH#774)
```

Comment:
RCTSlider is the paper subclass of `UISlider` that adds a lot of extra RN handling. I feel like this should be something called `RCTUISlider` that is a typedef on iOS, and a subclass of NSSlider on macOS? Otherwise this change is a deviation from how RN Core handles Slider